apps/evm_family/evm_sign_msg.c

```diff
@@ -200,11 +202,13 @@ static bool check_which_request(const evm_query_t *query,
 }
 
 static bool validate_initiate_query(evm_sign_msg_initiate_request_t *init_req) {
+  uint32_t size_limit = MAX_MSG_DATA_SIZE;
+
   switch (init_req->message_type) {
+    case EVM_SIGN_MSG_TYPE_SIGN_TYPED_DATA:
+      size_limit = MAX_MSG_DATA_TYPED_DATA_SIZE;
```

Comment:
the variable is initialized and not used in the check at line 220.

Reply:
updated